aws_robomaker_cloudwatch_publisher/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>aws_robomaker_cloudwatch_publisher</name>
+  <version>1.0.0</version>
+  <description>
+    AWS RoboMaker package containing utilities and nodes that facilitate pushing data and metrics to CloudWatch.
+  </description>
+  <license>Apache 2.0</license>
+  <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
+  <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>rostest</build_depend>
```

Comment:
We probably don't want to put a `boto3` dependency in here, but maybe a `requirements.txt` file and instructions in the README about it?

Reply:
I think you meant to comment on the setup.py? Should be fixed in latest commit.